src/vm/typehashingalgorithms.h

```diff
@@ -97,3 +97,94 @@ inline static int ComputeGenericInstanceHashCode(int definitionHashcode, int ari
     }
     return (hashcode + _rotl(hashcode, 15));
 }
+
+/*
+
+The below hash combining function is based on the xxHash32 logic implemented
```

Comment:
It seems we should put the license into THIRD-PARTY-NOTICES.TXT too.

Reply:
Its already in there. See License notice for xxHash.